meminfo.go

```diff
@@ -162,114 +214,179 @@ func parseMemInfo(r io.Reader) (*Meminfo, error) {
 	var m Meminfo
 	s := bufio.NewScanner(r)
 	for s.Scan() {
-		// Each line has at least a name and value; we ignore the unit.
 		fields := strings.Fields(s.Text())
-		if len(fields) < 2 {
-			return nil, fmt.Errorf("%w: Malformed line %q", ErrFileParse, s.Text())
-		}
+		var val, valBytes uint64
 
 		v, err := strconv.ParseUint(fields[1], 0, 64)
```

Comment:
How about we change this:

```suggestion
		val, err = strconv.ParseUint(fields[1], 0, 64)
```

Then we can drop the `val = v` below.

Reply:
Addressed in f7b4621